admin-manual/security/security.rst

```diff
@@ -50,16 +50,22 @@ the following into the command-line:
       FLUSH PRIVILEGES;"
 
 Once you've done this you can change Archivematica's MySQL database access
-credentials by editing these two files:
+credentials by editing your system files and changing the `CLIENT_USER` and
```

Comment:
Hi, the variable names are a bit different in each AM service.

Dashboard: 
- ARCHIVEMATICA_DASHBOARD_CLIENT_DATABASE
- ARCHIVEMATICA_DASHBOARD_CLIENT_HOST
- ARCHIVEMATICA_DASHBOARD_CLIENT_PASSWORD
- ARCHIVEMATICA_DASHBOARD_CLIENT_USER

MCPClient:
- ARCHIVEMATICA_MCPCLIENT_CLIENT_DATABASE
- ARCHIVEMATICA_MCPCLIENT_CLIENT_HOST
- ARCHIVEMATICA_MCPCLIENT_CLIENT_PASSWORD
- ARCHIVEMATICA_MCPCLIENT_CLIENT_USER

MCPServer:
- ARCHIVEMATICA_MCPSERVER_CLIENT_DATABASE
- ARCHIVEMATICA_MCPSERVER_CLIENT_HOST
- ARCHIVEMATICA_MCPSERVER_CLIENT_PASSWORD
- ARCHIVEMATICA_MCPSERVER_CLIENT_USER

Storage Service when using sqlite3:
- SS_DB_NAME
- SS_DB_HOST
- SS_DB_PASSWORD
- SS_DB_USER

Storage Service when using MySQL:
- SS_DB_URL  

For SS MySQL database see: [SS application-specific-environment-variables ](https://github.com/artefactual/archivematica-storage-service/blob/stable/0.15.x/install/README.md#application-specific-environment-variables) :

"The configuration of the database is also declared via environment variables. Storage Service looks up the SS_DB_URL environment string. If defined, its value is expected to follow the form described in the dj-database-url docs, e.g.: mysql://username:password@192.168.1.20:3306/storage_service. If undefined, Storage Service defaults to the django.db.backends.sqlite3 engine and expects the following environment variables to be defined"

Reply:
Please re-review!